daffodil-test/src/test/resources/org/apache/daffodil/section06/entities/charClassEntities.tdml

```diff
@@ -62,11 +62,12 @@
   </parserTestCase>
 
   <parserTestCase name="CarriageReturn" root="matrix"
-    model="charClassEntities.dfdl.xsd" description="CSV-style tests and use of %NL; - DFDL-6-045R"
-    roundTrip="twoPass">
+    model="charClassEntities.dfdl.xsd" description="CSV-style tests and use of %NL; - DFDL-6-045R">
 
     <document>
-      <documentPart type="text"><![CDATA[0,1,2,3,4
5,6,7,8,9
10,11,12,13,14]]></documentPart>
+      <documentPart type="text"><![CDATA[0,1,2,3,4
+5,6,7,8,9
+10,11,12,13,14]]></documentPart>
```

Comment:
git autocrlf might change this file.  Is this testing CR in TDML files? I don't think that really going to work without doing some special stuff to ensure git doesn't muck with things, or that future formatters don't change to a standard like LF.

I would say if we need to test loading files with CR/CRLF, we should create strings of data and load that. We really can't easily rely on files having the expected line endings.